src/Recruit.Storage.Client/Core/Mongo/MongoDbConventions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using MongoDB.Bson.Serialization.Conventions;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Esfa.Recruit.Storage.Client.Core.Mongo
+{
+    public static class MongoDbConventions
+    {
+        public static void RegisterMongoConventions()
+        {
+            var pack = new ConventionPack();
+            pack.Add(new CamelCaseElementNameConvention());
+            ConventionRegistry.Register("camelCase", pack, t => true);
```

Comment:
Does this mean the name of the field in Mongo will always be the same as in the codebase apart from different case? As in we can;t change it?

Reply:
Yep. At the demo we thought using conventions over a mapping for each property would be safer.  We can always add explicit mappings if we need them

Reply:
👍